### DISCUSSION TITLE
weighting property on relationship?

### DISCUSSION THREAD

**Discussion by Lkruitwagen** [opensupplyhub/supplychaindata.exchange#16] [General]
For some use cases we want more data/properties on `Relationships` right? E.g. % shareholding for ownership, total supplier volume in a given year, etc.

Is this the kind of thing that would go into an extension?

**Comment by wonderchook** [comment]
Yes, we are intentionally keeping the core very light but additional information could go into an extension.

**Comment by shuyag** [comment]
Going to convert this to a discussion, because I think there's a really interesting larger discussion here around the kinds of data we might want to pull in that illustrate different kinds of relationships -- thanks @Lkruitwagen!
